iohk/common/Utils.hs

```diff
@@ -112,3 +114,8 @@ jsonLowerStrip n = AE.genericToJSON $ AE.defaultOptions { AE.fieldLabelModifier
 s3Link :: Region -> BucketName -> ObjectKey -> Text
 s3Link region (BucketName bucket) (ObjectKey key) =
   mconcat [ "https://s3-", AWS.toText region, ".amazonaws.com/" , bucket, "/", key ]
+
+cdnLink :: HasCallStack => Environment -> ObjectKey -> Text
```

Comment:
what about making this an entry in the `config.yaml` so development clusters support updating
this code makes it harder to test the whole update system in a disposable cluster

Reply:
To be fair, this code doesn't really change the status quo -- we had `s3Link` to the same effect previously.

Reply:
I agree that your proposal is desirable, though!

Reply:
Done..